### PR TITLE
Style change 2.0

### PR DESCRIPTION
**Pull request by desh2608**
This is a corrected version of https://github.com/k2-fsa/icefall/pull/690.

* (107df3b) First pass: We apply black formatting without the `--experimental-string-processing` feature which was causing some files to get deleted entirely.
* (d31db01) Second pass: I manually went through all the modifications and corrected several of the sort:
```python
            f"expected sample rate: {expected_sample_rate}. "
            f"Given: {sample_rate}"
```
being converted to:
```python
           f"expected sample rate: {expected_sample_rate}. " f"Given: {sample_rate}"
```
and a small number of cases where inline comments made the formatting weird.

At the very least, these changes should not cause anything to break. If there is something I missed, I am hoping they can be corrected as people come across them in the future.

### CODE REVIEW

**Comment by desh2608** [comment]
@csukuangfj I would request you to please briefly check the files in the main `icefall/` directory since they are used in all recipes, to ensure there are no breaking changes. (You can skip the changes in `egs`)

File: .github/workflows/style_check.yml [hunk 1]
```diff
@@ -45,17 +45,18 @@ jobs:
 
       - name: Install Python dependencies
         run: |
-          python3 -m pip install --upgrade pip black==21.6b0 flake8==3.9.2 click==8.0.4
-          # See https://github.com/psf/black/issues/2964
-          # The version of click should be selected from 8.0.0, 8.0.1, 8.0.2, 8.0.3, and 8.0.4
+          python3 -m pip install --upgrade pip black==22.3.0 flake8==5.0.4 click==8.1.0
```
Comment (csukuangfj) [inline .github/workflows/style_check.yml:48]: Could you also update
https://github.com/k2-fsa/icefall/blob/master/docs/source/contributing/code-style.rst

**Comment by csukuangfj** [comment]
Thanks! Looks good to me. Left a minor comment.

File: .github/workflows/style_check.yml (hunk 1 above)
Comment (desh2608) [reply .github/workflows/style_check.yml:48]: Done!

**Comment by desh2608** [comment]
> Thanks! Looks good to me. Left a minor comment.

I think it's ready for merge?

**Pull request merged**